### ISSUE TITLE
1.27.0: netlify host with tracker script name environment variable

### ISSUE THREAD

**Issue by duboiss** [umami-software/umami#1008]
Since v1.27.0, my custom tracker script name is not available on Netlify but still works on Heroku.
Its the same repository with the same environments variables.

<img width="632" alt="image" src="https://user-images.githubusercontent.com/29716703/156588804-66d35a57-c846-4ef4-853b-1a40a00abac3.png">

Without env variable, everything work well with `umami.js` file (which always seems to be accessible, even with the environment variable).

https://portfolio-unami.herokuapp.com/statistics.js
https://unami.duboiss.fr/statistics.js

I think its related to #731  & #977

**Comment by mikecao** [comment]
I'm not that familiar with Netlify, but I'm guessing it can't run the new Next.js middleware, which is how the tracker script name is handled. We may need to come up with a static solution for Netlify.

**Comment by mikecao** [comment]
No schema changes were made.

**Comment by AFilthyLeaf** [comment]
I have the same issue as this but on Heroku! Not sure what could be causing it.

**Comment by christopherpickering** [comment]
I did a few prints to try and debug as well. I set the env variable to "something_cool".

Modified function:
![image](https://user-images.githubusercontent.com/17788706/158652801-64ea5e5e-8b72-4fa9-900e-e1b7eb535cbf.png)

Netlify logs:
![image](https://user-images.githubusercontent.com/17788706/158652981-42397d73-4420-4123-916e-d5e8e9c6a706.png)

And when running locally with `npm run dev`

![image](https://user-images.githubusercontent.com/17788706/158653196-5a792186-56af-4fb2-b9d5-cdd759f2c62b.png)

But when I check the network logs on the netlify build, it looks like it tries to get the umami.js properly. But somehow returns 404 instead of 200.

I wonder if the href of the url object needs to be updated as well? I'll test it out.
